dgv/monitoring/DAG-notification.py

```diff
@@ -74,6 +76,17 @@ def check_new(ti, **kwargs):
                 for spam in SPAM_WORDS
                 for field in [item['name'], item['description']]
             ])
+        # checking for potenital duplicates in organization creation
+        mydict['duplicated'] = False
+        if templates_dict["type"] == 'organizations':
+            slug = item["slug"]
+            if re.search(duplicate_slug_pattern, slug) is not None:
+                suffix = re.findall(duplicate_slug_pattern, slug)[0]
+                original_orga = slug[:-len(suffix)]
+                test_orga = requests.get(f"https://data.gouv.fr/api/1/organizations/{original_orga}/").json()
+                # only considering a duplicate if the original slug is taken
+                if not test_orga.get('message', False) and not test_orga.get("deleted", False):
```

Comment:
I'm not a fan of looking at `message` instead of using explicit status code (404 or 410). You wouldn't get a `deleted`  property anyway if you receive a 410 I think